src/dataflow/source/kinesis.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::server::{TimestampChanges, TimestampHistories};
+use dataflow_types::{Consistency, ExternalSourceConnector, KinesisSourceConnector, Timestamp};
+use timely::dataflow::{Scope, Stream};
+
+use super::util::source;
+use super::{SourceStatus, SourceToken};
+use expr::SourceInstanceId;
```

Comment:
If you go back through CI for something else, these imports should be sorted like so:

```rust
use timely::dataflow::{Scope, Stream};

use expr::SourceInstanceId;
use dataflow_types::{Consistency, ExternalSourceConnector, KinesisSourceConnector, Timestamp};

use crate::server::{TimestampChanges, TimestampHistories};
use super::util::source;
use super::{SourceStatus, SourceToken};
```

The groups are: stdlib (not present here), third-party crates, first-party crates, this crate.